randovania/games/samus_returns/logic_database/Area 3 - Factory Interior.txt

```diff
@@ -604,13 +600,12 @@ Extra - asset_id: collision_camera_027
       Any of the following:
           Single-wall Wall Jump (Beginner) or Super Jump (Intermediate) or Climb Rooms Vertically (High Jump)
           Morph Ball and Unmorph Extend (Beginner)
-          Stand on Frozen Enemy (Beginner) and Fully Freeze Enemy
```

Comment:
Do the enemies not appear anymore? If they only appear after Gamma is killed, cant you gate them after the "After Gamma killed" event?

Reply:
Due to how we fixed the multi room Gammas to be properly disabled, the enemies don't exist anymore. It could involve some event node placed outside the arena, but at that point you would have the items to leave so its pointless logic.